enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java

```diff
@@ -203,10 +204,9 @@ else if ( role.equals( HighAvailabilityModeSwitcher.SLAVE ) )
                 {
                     HighAvailabilityMemberState oldState = state;
                     state = state.slaveIsAvailable( context, instanceId, roleUri );
-                    logger.debug( "Got slaveIsAvailable(" + instanceId + "), " +
-                            "moved to " + state + " from " + oldState );
                     final HighAvailabilityMemberChangeEvent event = new HighAvailabilityMemberChangeEvent( oldState,
                             state, instanceId, roleUri );
+                    logger.debug( "Got slaveIsAvailable(" + instanceId + "), " + "changed " + oldState + " -> " + state );
```

Comment:
I think if we want to change the verb from 'moved' to 'changed' then we should do it in the other places that use 'moved' too:

e.g

https://github.com/neo4j/neo4j/blob/master/enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/SwitchToMaster.java#L98
https://github.com/neo4j/neo4j/blob/master/enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/SwitchToSlave.java#L235

Use case for that is easily searching for HA state changes in messages.log i.e.

```
grep --color -i -rn "moved to" db/2/messages.log
```

WDYT?